src/main/java/Utilities/Ui.java

```diff
@@ -0,0 +1,148 @@
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println((i + 1) + ". " + tasks.get(i));
+        }
+        System.out.println("_____________________________________");
+    }
+
+    /**
+     * Displays a message indicating that no tasks were found.
+     */
+    public void noTask() {
+        System.out.println("_____________________________________");
+        System.out.println("No Task Found");
+        System.out.println("_____________________________________");
+    }
+
+    /**
+     * Displays a message indicating whether a task was marked as done or not done.
+     *
+     * @param tasks    The list of tasks.
+     * @param index    The index of the task to mark.
+     * @param isToMark true if marking as done, false if marking as not done.
+     */
+    public void markTask(ArrayList<Task> tasks, int index, Boolean isToMark) {
+        System.out.println("_____________________________________");
+        if (isToMark) {
+            System.out.println("Nice! I've marked this task as done:");
+        } else {
+            System.out.println("OK, I've marked this task as not done yet:");
+        }
+        System.out.println("  " + tasks.get(index - 1));
+        System.out.println("_____________________________________");
+    }
+
+    /**
+     * Displays a message indicating whether a task was added or removed.
+     *
+     * @param tasks The list of tasks.
+     * @param task  The task that was added or removed.
+     * @param isAdd true if the task was added, false if removed.
+     */
+    public void alterTask(ArrayList<Task> tasks, Task task, Boolean isAdd) {
+        System.out.println("_____________________________________");
+        if (isAdd) {
+            System.out.println("Got it. I've added this task:");
+        } else {
+            System.out.println("Noted. I've removed this task:");
+        }
+        System.out.println("  " + task);
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        System.out.println("_____________________________________");
+    }
+
+    /**
+     * Displays a default message when the user input is not recognized.
+     */
+    public void defaultMsg() {
+        System.out.println("_____________________________________");
+        System.out.println("I'm sorry, but I don't know what that means");
+        System.out.println("_____________________________________");
+    }
+
+    /**
+     * Displays a farewell message when the user exits the application.
+     */
+    public void endMsg() {
+        System.out.println("_____________________________________");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("_____________________________________");
+    }
+
+    /**
+     * Displays a greeting message to the user.
+     *
+     * @param logo The name or logo of the application.
+     */
+    public void greetMsg(String logo) {
```

Comment:
Use standard words and avoid abbreviations unless they are well understood. Replace Msg with Message